<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerwidget</class>
 <widget class="QWidget" name="registerwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>registerwidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px</string>
  </property>
  <widget class="QFrame" name="frmBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>191</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/VlcPlayer/D:/苏嵌合作/视频传输/图片/背景2.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(218, 218, 218);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWelcome">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/VlcPlayer/D:/苏嵌合作/图片/注册.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="tlbUName">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="tlbPassword">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lePassword">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leUName">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="tlbPassword_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>密码确认</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lePassword_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnRegiste">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);
font: 11pt &quot;微软雅黑&quot;;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vlcplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>registerwidget</receiver>
   <slot>CloseRegisterForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRegiste</sender>
   <signal>clicked()</signal>
   <receiver>registerwidget</receiver>
   <slot>Regist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CloseRegisterForm()</slot>
  <slot>Regist()</slot>
 </slots>
</ui>
